platforms/shared/configuration/roles/setup/certmanager/tasks/install_certmanager.yaml

```diff
@@ -0,0 +1,27 @@
+---
+##Checks if cert-manager is installed 
+- name: check cert-manager deployment status
+  kubernetes.core.k8s_info:
+    kubeconfig: "{{ kubeconfig_path }}"
+    kind: Deployment
+    name: cert-manager
+    namespace: cert-manager
+  register: certmanager
+  tags:
+    - molecule-idempotence-notest
+
+##installs cert-manager if its not installed
+- name: Install cert-manager
+  kubernetes.core.helm:
+    kubeconfig: "{{ kubeconfig_path }}"
+    name: cert-manager
+    namespace: cert-manager
+    state: present
+    chart_ref: "{{ playbook_dir }}/../../shared/charts/cert-manager"
```

Comment:
better to package the cert-manager chart and use it

Reply:
packaged